### PR TITLE
remove image urls

### PR DESCRIPTION
**Pull request by marhalpert**
@nguyenchr @eddiemcleanlux

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -83,16 +83,12 @@ function getRegionNamesAndCode() {
   })
 }
 
-function getRegionFlagUrlsByCode(regionCode, cloudName) {
-  if (!regionCode || !cloudName) {
+function getRegionFlagUrlsByCode(regionCode) {
```
Comment (eddiemcleanlux) [inline index.js]: Function needs to be renamed.

**Comment by eddiemcleanlux** [comment]
I guess if we've removed the need for a cloud name argument, the flag IDs might as well be put into the region objects in the main `region-data.js` file.

**Comment by marhalpert** [comment]
I don't think we need them there, I actually wanted to make a PR to move the telephones out

**Comment by eddiemcleanlux** [comment]
Also keep semver in mind, technically this is a breaking change even though the thing it's breaking was only merged three hours ago.

**Comment by nguyenchr** [comment]
I kind of like having everything in region-data
otherwise every client needs to know how to stitch together this data
but I'm not too fussed either way

**Comment by marhalpert** [comment]
no worries 2 against 1 :)

**Pull request merged**